packages/instantsearch.js/src/widgets/index/index.ts

```diff
@@ -652,6 +652,12 @@ const index = (widgetParams: IndexWidgetParams): IndexWidget => {
           new algoliasearchHelper.SearchParameters(indexInitialResults.state),
           indexInitialResults.results
         );
+        if (
+          '_rawContent' in indexInitialResults &&
+          indexInitialResults._rawContent
+        ) {
+          results._rawResults = indexInitialResults?._rawContent as any;
+        }
```

Comment:
without a change in the initial results this will always be undefined

Reply:
if you don't need it for the dashboard I'd rather leave it without change in the index widget.

The code also seems wrong, why are you overriding rawResults with rawContent?

Reply:
> why are you overriding rawResults with rawContent

this is definitely a mistake

> if you don't need it for the dashboard I'd rather leave it without change in the index widget

works for me 🙂